std/experimental/ndslice/slice.d

```diff
@@ -567,6 +567,19 @@ slice(T,
 }
 
 /// ditto
+auto slice(T,
+    Flag!`replaceArrayWithPointer` replaceArrayWithPointer = Yes.replaceArrayWithPointer,
+    size_t N)(auto ref in size_t[N] lengths, auto ref T init)
```

Comment:
you already do the same for `makeSlice` - https://github.com/dlang/phobos/blob/master/std/experimental/ndslice/slice.d#L627